app/_src/gateway/kong-manager/workspaces.md

```diff
@@ -75,45 +75,34 @@ navigate to the new Workspace's dashboard.
 
 ## Delete a workspace
 
-### Wipe workspace data
 To delete a workspace, *all data* must first be deleted from the workspace.
```

Comment:
```suggestion
{:.important}
> **Caution**: The following steps will also delete **all entities in the workspace**. Make sure that this is something you want to do before proceeding.

```
Feel free to rephrase this, but since we're showing the force delete as a default now, I think we should make this into a caution.